internal/go-prompt/document.go

```diff
@@ -198,6 +199,23 @@ func (d *Document) FindEndOfCurrentWord() int {
 	return len(x)
 }
 
+func (d *Document) FindPreviousWordStart() int {
```

Comment:
Could you add a test for this method?

Reply:
Thanks, added at https://github.com/hazelcast/hazelcast-commandline-client/pull/64/commits/60b613b79be4e727690ba65811c2589193df16ce